src/server.ts

```diff
@@ -1,8 +1,11 @@
 import { example } from "./rest/example";
 import express from "express";
+import * as dotenv from "dotenv";
```

Comment:
minor nitpick but this can be just `import dotenv from "dotenv"`

Reply:
You're right!